### ISSUE TITLE
NTR: Delayed passage of meconium (GI, feeding, vomiting & dehydration)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7893]
New term request Delayed passage of meconium (brumme_220603120805): >48 hours after birth  UMLS: 0349500

**Comment by pnrobinson** [comment]
@MickeySegal
I am trying to decide what parent term this term should have. It is no longer a prenatal anomaly although it is closely related. 

def: Failure to pass meconium (the first stool following birth) in the first 48 hours of life.
Comment: Failure to pass meconium combined with progressive abdominal distention, refusal to feed and vomiting of bilious intestinal contents are the classic clinical signs of intestinal obstruction in neonates. The first stool is passed within 24 hours of birth in 99 percent of healthy full-term infants and within 48 hours in all healthy full-term infants.
PMID:10569507

**Comment by MickeySegal** [comment]
That seems good.

I see that HP:0001197 includes birth, but perinatal goes out days or weeks from that.

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000224
- label: Delayed passage of meconium
- parents: [Abnormality of prenatal development or birth (HP:0001197)](https://hpo.jax.org/app/browse/term/HP:0001197)
- pmid: PMID:10569507
- def: Failure to pass meconium (the first stool following birth) in the first 48 hours of life.
- comment: Failure to pass meconium combined with progressive abdominal distention, refusal to feed and vomiting of bilious intestinal contents are the classic clinical signs of intestinal obstruction in neonates. The first stool is passed within 24 hours of birth in 99 percent of healthy full-term infants and within 48 hours in all healthy full-term infants.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7893
- synonyms:
